egs/wsj/s5/steps/online/nnet2/extract_ivectors_online.sh

```diff
@@ -76,6 +75,12 @@ done
 
 # Set various variables.
 mkdir -p $dir/log $dir/conf
+num_cmn_offsets=-1  # If > 0, it uses offsets.scp and generates iVector for 
+                    # all random offsets version of inputs.
+if [ -f $data/offsets.scp ]; then
+  num_cmn_offsets=`feat-to-len --print-args=false scp:"head -n 1 $data/offsets.scp |"` 
+  nj=$[$nj*$num_cmn_offsets] 
```

Comment:
please remove this code that adjusts 'nj'.  The 'nj' option means "run this many jobs", scripts should never auto-adjust.  Just use more jobs in the top-level script if it's too slow.